x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/plugin/TransportEqlSearchAction.java

```diff
@@ -95,7 +95,8 @@ public void execute(EqlSearchRequest request, EqlSearchTask task, ActionListener
     @Override
     public EqlSearchResponse initialResponse(EqlSearchTask task) {
         return new EqlSearchResponse(EqlSearchResponse.Hits.EMPTY,
-            threadPool.relativeTimeInMillis() - task.getStartTime(), false, task.getExecutionId().getEncoded(), true, true);
+            TimeValue.nsecToMSec(task.getStartTimeNanos()) - threadPool.relativeTimeInMillis(), false, task.getExecutionId().getEncoded(),
```

Comment:
getStartTimeNanos is obtained from System.nanoTime(). So, would something like `TimeValue.nsecToMSec(System.nanoTime() - task.getStartTimeNanos())` make more sense in this case?

Reply:
Thanks, indeed, _that_ makes sense.